<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_changepwd</class>
 <widget class="QDialog" name="User_changepwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>288</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>288</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>288</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>旧密码：</string>
   </property>
   <property name="buddy">
    <cstring>oldlineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>131</y>
     <width>120</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>确认新密码：</string>
   </property>
   <property name="buddy">
    <cstring>againlineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
   <property name="buddy">
    <cstring>newlineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="againlineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="newlineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldlineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>User_changepwd</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
